app/controllers/org_admin/users_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module OrgAdmin
+
+  class UsersController < ApplicationController
+
+    after_action :verify_authorized
+
+    def edit
+      @user = User.find(params[:id])
+      authorize @user
+      @plans = Plan.active(@user).page(1)
+      render "org_admin/users/edit",
+             locals: { user: @user,
```

Comment:
Don't bother changing your PR here @johnpinto1, I just wanted to point out that this is a pattern that got introduced some time ago and we've been carrying it forward. There's no need for us to assign these `locals` when we've already defined instance variables. You can just reference `@user` directly in the view.

I'll add it to the agenda to discuss in our next dev call.